src/main/scala/tech/cryptonomic/conseil/routes/openapi/OpenApiDoc.scala

```diff
@@ -51,7 +51,8 @@ object OpenApiDoc extends DataEndpoints
       documentation = invalidDocs.getOrElse(""),
       content = Map(
         "application/json" -> MediaType(None),
-        "text/csv" -> MediaType(None)
+        "text/csv" -> MediaType(None),
+        "text/plain" -> MediaType(None)
```

Comment:
I don't see this content type in akka 😉

Reply:
you should make your own I guess, I don't think `sql` is a common mime-type because I don't think anyone wants to execute that automatically while reading the content from the browser...

yet: https://www.iana.org/assignments/media-types/application/sql